designs/node-labels.md

```diff
@@ -0,0 +1,81 @@
+# Add support for node labels
+
+## Introduction
+
+**Problem:** Within an EKS-A cluster, the user might want to run specific workloads only on certain nodes.
+A great way to do this is by assigning labels to the desired nodes.
+However, there is currently no way for users to add labels to their nodes.
```

Comment:
super nit: it's actually possible to add labels to nodes manually, post cluster creation. The problem right now is those labels are not persistent. If the node or machine are deleted or if the user upgrades the cluster (which rolls new machines hence new nodes) those labels will be lost and will need to be added again.

Reply:
Thanks for pointing that out! I think that's pretty important to note so I'll update the doc to mention this